CHANGELOG.md

```diff
@@ -5,6 +5,12 @@
 
 All notable changes to this project will be documented in this file.
 
+## [1.5.1] - 2025-07-21
+
+### Fixed
+
+- Fixed a bug on apple-darwin where the xz library wasn’t found during dynamic loading. Resolved by statically linking the library.
```

Comment:
This is technically not correct, the bug was that we depended on the user having it installed, and if they did, there was no issue. The problem only occured if they did not have it installed, right?

Reply:
that was my understanding, `where` -> `when` ?  
Will improve the description